l10n_cl_invoicing_policy/models/res_partner.py

```diff
@@ -30,14 +30,12 @@ def _default_invoicing_policy(self):
 
     @api.constrains('is_company', 'invoicing_policy')
     def check_invoicing_policy(self):
-        if self.invoicing_policy == 'ticket':
-            if self.is_company:
-                raise UserError(_('The invoicing policy Ticket only applies '
-                                  'to individuals.'))
-        else:
-            if not self.is_company:
-                raise UserError(_('The selected invoicing policy only applies'
-                                  ' to companies.'))
+        if self.invoicing_policy == 'ticket' and self.is_company:
+            raise UserError(_('The invoicing policy Ticket only applies '
+                              'to individuals.'))
+        if self.invoicing_policy != 'ticket' and not self.is_company:
+            raise UserError(_('The selected invoicing policy only applies'
+                              ' to companies.'))
```

Comment:
@dreispt If the invoicing policy is `invoice` and the record is an address or contact of a company (is_company = False), it will raise the error although it is a legitimate case.